ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -427,8 +427,10 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
   setDebugModeEnabled: (enabled: boolean) =>
     dispatch(setDebugModeEnabled(enabled)),
   dispatchSessionExpired: () => dispatch(sessionExpired()),
-  setPagoPAQAEnabled: (enabled: boolean) =>
-    dispatch(pagoPaQAEnvironmentEnabledSuccess({ isPagoPAQAEnabled: enabled })),
+  setPagoPATestEnabled: (enabled: boolean) =>
```

Comment:
minor styling note, you if rename `enabled` to `isPagoPATestEnabled`, you can make the next param pass shorter:

 ```ts
setPagoPATestEnabled: (isPagoPATestEnabled: boolean) =>
dispatch(
      pagoPaTestEnvironmentEnabledSuccess({ isPagoPATestEnabled })
    )
```

Reply:
nice!